File: Team C (Season 2)/Week 1/Greed/42842. 카펫/카펫.swift
import Foundation

func solution(_ brown:Int, _ yellow:Int) -> [Int] {
    var t : Int = (brown - 4)/2
    var answer : [Int] = []
    
    for i in 1...t {
        var width : Int = t - i
        var height : Int = i
        if width * height == yellow && width >= height {
            answer = [width + 2, height + 2]
        }
    }
    
    
    return answer
}

File: Team C (Season 2)/Week 1/Greed/42842. 카펫/카펫.py
def solution(brown, yellow):
    t = int((brown - 4)/2)
    
    for i in range(2 * t):
        width = t - i
        height = t - width
        if width * height == yellow and width >= height:
            answer = [width + 2, height + 2]
            
    return answer